WordPress/gutenberg
# Image block: Disable flaky image tests related to Interactivity API

## What?

This PR disables the flaky image tests that have recently become flaky for now while we work on solution (being discussed [here](https://github.com/WordPress/gutenberg/pull/52442)).

## Why?

We want to avoid slowdowns in the deployment process.

## How?

It comments out the image tests related to the Interactivity API.

## Testing Instructions




N/A

### Testing Instructions for Keyboard

Run `npm run test:e2e:playwright` — see that the `Image - interactivity` tests do not run.
